config.py

```diff
@@ -16,9 +16,11 @@ def __init__(self):
         self.backgroundImage = pygame.image.load(resource_path(
             os.path.join('assets', 'graphics', 'background-black-wide.png')))
 
-        windows_user = ctypes.windll.user32
-        self.monitor_size = (windows_user.GetSystemMetrics(0),
-                             windows_user.GetSystemMetrics(1))
+        self.CANVAS = pygame.display.set_mode(
+            (self.WIDTH, self.HEIGHT), pygame.RESIZABLE)
+
+        self.monitor_size = pygame.display.Info().current_w, pygame.display.Info().current_h
+        # print(pygame.display.Info())
```

Comment:
Can we remove this comment

Reply:
Hey! The comment was in the code before and I actually used it to make the change so i was thinking maybe its there for when people try to change something within the code, that is why i left it for now. I don't really know if that was the initial intention of the creator :D

Reply:
@naddoska can remove the comment, I must have forgotten about it. @nkitsaini thanks for pointing out

Reply:
@Mayank0255  I fixed it :)